package.json

```diff
@@ -46,7 +46,7 @@
     "copy-webpack-plugin": "4.0.1",
     "coveralls": "^2.11.15",
     "css-loader": "^0.26.0",
-    "d3": "^4.4.1",
+    "d3": "^4.9.1",
```

Comment:
So.. current dependency constraints:

   * kubernetes-topology-graph needs >=3.5.0, <=3.5.5 - but I assume we will drop `kubernetes-topology-graph` with the new topology component, right?
   * `patternfly-sass` needs ~3.5.17 - so we'll need to update patternfly to a version compatible with d3 4
   * `patternfly-sass` needs **c3** ~0.4.11, which depends on d3 ~3.5.0 - so, we'll need to update patternfly to a version compatible with new enough c3 - but even the newest has that dependnecy, so either the dependency is obsolete or we can't move to d3 4 yet